Don't force BOOST_FILESYSTEM_VERSION=2, this is not supported in boost>=1.50

--- CMakeLists.txt~	2014-04-29 12:06:55.000000000 +0200
+++ CMakeLists.txt	2014-06-19 08:16:36.244048000 +0200
@@ -47,8 +47,6 @@
 # Add Boost path to include directories.
 INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIRS})
 
-# Make sure Boost.Filesystem v2 is used.
-ADD_DEFINITIONS(-DBOOST_FILESYSTEM_VERSION=2)
 
 ADD_SUBDIRECTORY(src)
 ADD_SUBDIRECTORY(examples)
